examples/dd.qasm

```diff
@@ -5,9 +5,9 @@
 OPENQASM 3.0;
 include "stdgates.inc";
 
-length start_stretch = -0.5 * lengthof({x %0;}) + stretch
-length middle_stretch = -0.5 * lengthof({x %0;}) - 5 * lengthof({y %0;} + stretch
-length end_stretch = -0.5 * lengthof({y %0;}) + stretch
+length start_stretch = -0.5 * lengthof({x %0;}) + stretch;
```

Comment:
I do not think `stretch` should be allowed to be defined w/ out a variable like this. I believe this should only occur for `stretchinf`. Instead we should have a variable defined above `stretch s;` and then use that